MIPs/mip-1.md

```diff
@@ -1,7 +1,7 @@
 ---
 MIP: 1
 Title: Extend `wallet_watchAsset` to Support ERC-721 and ERC-1155 Tokens (NFTs)
-Status: Last Call
+Status: Experimental
```

Comment:
According to our process guide, the next status should be `Accepted`. Then once it's moved to `Implemented` the Stability will be changed from n/a to `Experimental`.

Reply:
done [here](https://github.com/MetaMask/metamask-improvement-proposals/pull/20/commits/3da6b6c4cc5e9e15fcabca77767edfec44c8f92b)